corehq/sql_db/util.py

```diff
@@ -297,7 +297,10 @@ def get_standby_delays_by_db():
     return ret
 
 
-@quickcache(['dbs'], timeout=STALE_CHECK_FREQUENCY, skip_arg=lambda *args: settings.UNIT_TESTING)
+@quickcache(
+    ['dbs'], timeout=STALE_CHECK_FREQUENCY, skip_arg=lambda *args: settings.UNIT_TESTING,
+    memoize_timeout=STALE_CHECK_FREQUENCY, session_function=None
+)
```

Comment:
the timing is same for `timeout` and `memoize_timeout`, that means it stays in memory and also in the cache, hence on disk for the same time? hence the cache won't ever get hit?

Reply:
Could not find it on quickcache repo, why is `session_function=None` needed?

Reply:
`session_function` is set here in the default quickcache: https://github.com/dimagi/commcare-hq/blob/cc93bc6ab718bde08e8d16e374d5cc25eff94958/corehq/util/quickcache.py#L45

Setting it to None will mean that local memory cache can be used across requests / tasks.

Reply:
> the timing is same for timeout and memoize_timeout, that means it stays in memory and also in the cache, hence on disk for the same time? hence the cache won't ever get hit?

having memoize_timeout the same as the cache timeout just means that they should both timeout at the same time.

Reply:
Thank you for the response Simon. In such a case, will the cache be used ever?

Reply:
yes, the cache will still be used since there are multiple processes